src/api/globals.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @prettier
+ */
+const config = require('../config');
+const util = require('./util');
+
+module.exports = {
+    /**
+     * #### require(name)
+     *
+     * Load an npm package (the real "require" has its own cache).
+     */
+    // TODO: Consider supporting macros in sub-directories
+    require: util.createRequire(config.macrosDirectory),
+};
```

Comment:
Oh, this is interesting that you have to do it that way. In that case, I'd prefer to just define globalsAPI directly in environment.js so that we don't need this special utility function and to reduce the risk of the change.  Sorry to ask you to change it and then change it back.

The risk of subtle behavior differences between the "real" require and the function returned by createRequire() is what I worry about, since we don't have a robust test suite.